ssaHAL/ssa/interfaces/runtime.py

```
@@ -0,0 +1,57 @@
+from .connector import SSAConnector
+from .affordance_handler import AffordanceHandler
+
+
+class SSARuntime(SSAConnector, AffordanceHandler):
+    """Base class defining the SSARuntime interface."""
+
+    def __init__(self, config, extra_config_template={}):
```

Ruff (0.8.2)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)